src/main/java/main/model/dto/ProjectUserDto.java

```diff
@@ -29,22 +29,26 @@
     private Integer viewer;
 
     public boolean isAdmin(){
-        return admin > 0;
+        return isPermissionTrue(admin);
     }
 
     public boolean isManager(){
-        return manager > 0;
+        return isPermissionTrue(manager);
     }
 
     public boolean isEngineer(){
-        return engineer > 0;
+        return isPermissionTrue(engineer);
     }
 
     public boolean isViewer(){
-        return viewer > 0;
+        return isPermissionTrue(viewer);
     }
 
     public boolean isEditor() {
-        return admin > 0 || manager > 0 || engineer > 0;
+        return isAdmin() || isManager() || isEngineer();
+    }
+
+    private boolean isPermissionTrue(Integer permission){
```

Comment:
can we have only one implementation of isPermissionTrue?
now I see two in different files

Reply:
fixed